ci/cloudbuild/build.sh

```diff
@@ -190,31 +196,39 @@ if [[ "${LOCAL_FLAG}" = "true" ]]; then
     die "Only one of --local, --docker, or --cloud may be specified"
   fi
 
-  # Prints links to the log files for the current build. These can be useful to
-  # copy-n-paste into issues. The GCB link will require auth but the raw link
-  # may be publicly accessible.
-  io::log_h1 "Log Links"
-  printf "GCB: %s\n" "${CONSOLE_LOG_URL:-none}"
-  printf "Raw: %s\n" "${RAW_LOG_URL:-none}"
-
   function mem_total() {
     awk '$1 == "MemTotal:" {printf "%0.2f GiB", $2/1024/1024}' /proc/meminfo
   }
   function google_time() {
     # Extracts the time that Google thinks it is.
     curl -sI google.com | sed -n 's/Date: \(.*\)\r/\1/p'
   }
-  io::log_h1 "Machine Info"
-  printf "%10s %s\n" "host:" "$(date -u --rfc-3339=seconds)"
-  printf "%10s %s\n" "google:" "$(date -ud "$(google_time)" --rfc-3339=seconds)"
-  printf "%10s %s\n" "kernel:" "$(uname -v)"
-  printf "%10s %s\n" "os:" "$(grep PRETTY_NAME /etc/os-release)"
-  printf "%10s %s\n" "nproc:" "$(nproc)"
-  printf "%10s %s\n" "mem:" "$(mem_total)"
-  printf "%10s %s\n" "term:" "${TERM-}"
-  printf "%10s %s\n" "gcc:" "$(gcc --version 2>&1 | head -1)"
-  printf "%10s %s\n" "clang:" "$(clang --version 2>&1 | head -1)"
-  printf "%10s %s\n" "cc:" "$(cc --version 2>&1 | head -1)"
+  if [[ "${TRIGGER_TYPE}" != "manual" ]]; then
+    # Prints links to the log files for the current build. These can be useful
+    # to copy-n-paste into issues. The GCB link will require auth but the raw
+    # link may be publicly accessible. In manual builds this information is
+    # never useful (both are "none").
+    io::log_h1 "Log Links"
+    printf "GCB: %s\n" "${CONSOLE_LOG_URL:-none}"
+    printf "Raw: %s\n" "${RAW_LOG_URL:-none}"
+  fi
+
+  if [[ "${TRIGGER_TYPE}" != "manual" || "${VERBOSE_FLAG}" == "true" ]]; then
+    # Prints information about the machine and compiler. In manual builds this
+    # information is almost never useful.
```

Comment:
I think this info can be useful on manual builds, because they all run in Docker containers so the OS and compilers versions and stuff are not always obvious. My opinion would be to continue printing this, unless others think this is too noisy during manual builds.

Reply:
Ooops, you triggered the merge before we could discuss, my trap has sprung, mua ha ha.

Obviously **I** think it is too noisy, most of the time I am running a build locally this information gooes by unread.  If I need it I can run again with `--verbose`.  But we can change it back, adding this to the team meeting agenda.